R/adae.R

```diff
@@ -1,5 +1,6 @@
-#' Adverse Event Analysis
+#' Dataset adae
 #'
+#' Adverse Events Analysis
```

Comment:
I think the label for each of the datasets (using this one as an example) have been reversed to the old one. Is there a step we missed

Reply:
I applied label for the description only in  [create_adams_data.R](https://github.com/pharmaverse/pharmaverseadam/blob/5304a9143dc338cfa66c1094f88cd6f90e877ccb/data-raw/create_adams_data.R#L37-L39R)- 
![image](https://github.com/pharmaverse/pharmaverseadam/assets/47867131/4e6d77a8-8ac9-4ed5-a3a5-2353e619cb22)
vs

https://pharmaverse.github.io/pharmaverseadam/reference/adae.html has labels for both title and description 
![image](https://github.com/pharmaverse/pharmaverseadam/assets/47867131/a75f34cf-890d-498c-a43f-964e640ab4d9)


Let me know your preference, I can do a quick update if needed :)

Reply:
I think the first one is better without 'Dataset'
<img width="356" alt="Screenshot 2024-05-29 at 1 59 56 PM" src="https://github.com/pharmaverse/pharmaverseadam/assets/112789549/3a533ab6-1064-49b4-975d-07de4ee7fbdf">